### DISCUSSION TITLE
CF client performance issues

### DISCUSSION THREAD

**Discussion by matejkarasek** [bitcraze/discussions#1513] [Q&A] [answered]
Hi,
we pip installed CF client on a new laptop (Win 11) and, once we connect to a crazyflie over Crazyradio 2, it starts freezing and becoming unresponsive randomly. The laptop has enough resources (latest gen i7, 32 GB of RAM, dedicated GPU)
Also, this was much worse with the newest cfclient version (2024.7.1), it improved to some extent when downgrading to 2024.2.1 (we did not test 2024.3 nor 2024.7)

We noticed this happens especially when the laptop is running of battery, but even when powered from AC this sometimes happens. So it might be related to some power management thing...

However, on the same laptop, we use cflib to control a swarm of 4 Bolt-based drones simultaneously with no issues.

Any ideas what could be wrong?

Thanks,
Matej

**Comment by matejkarasek** [comment]
@JVprodC FYI

**Comment by knmcguire** [comment, marked as answer]
Hi Matej,

Yes we have seen this as well but have made no issue open for this. I have it on my windows whenever I'm in battery saving mode. Apparently the throttling of the gpu also causes some elements of QT6 to be much slower, which has difficulties with catching up...

I have opened an ticket for this and we will discuss it at the next triage: https://github.com/bitcraze/crazyflie-clients-python/issues/723

**Comment by matejkarasek** [comment]
Ok, thanks, will keep an eye on the Github issue!
